man/deepsea-stage-3.7

```diff
@@ -0,0 +1,174 @@
+.TH deepsea 7
+.SH NAME
+DeepSea stage 3 \- a deeper insight
+.SH DESCRIPTION
+.B DeepSea Stage 3 (Deploy)
+is a Salt orchestration that deploys the core services of ceph
+.RE
+.PD
+.SH Preface
+.PP
+After configuration and setup stages have been completed successfully we are ready to actually deploy and install ceph.
+
+.B Validation:
+.RS
+DeepSea performs a set of system validation tests before running the ramainder of the stage.
+
+* DEV_ENV
+  This option tells you if a Developer Environment is expected. Setting this option bypasses some validations that 
+  would lead to insufficient replication or data insafety.
+  Do not set this option in production.
+
+.B Disabled validations:
+  * monitors
+  * storage
+  * mgrs
+
+* firewall 
+  Verify if the neccessary ports for ceph are open in the firewall.
```

Comment:
No ports are actually checked.  The default chains are checked for default rules.

Reply:
So we are actually checking the negative -> if the ports are not closed.